### DISCUSSION TITLE
How can I export current pointer and data?

### DISCUSSION THREAD

**Discussion by janfeitsma** [facontidavide/PlotJuggler#592] [Q&A] [answered]
I use the binary .AppImage, version 3.2.0, without ROS.

I would like to connect a custom external tool to the current data from the PlotJuggler 'time pointer'. Similar to impressive rviz/ROS demo.

The CSV publisher might be what I need, but I cannot get it to work; the dialog is not intuitive to me and I never managed to get the CSV file contents to change along with my GUI interaction. 

Can I use standard CSV publisher, and if so, how?
Or should I use a different plugin; perhaps even write my own?

**Comment by facontidavide** [comment, marked as answer]
The CSV publisher is limited and... not really a publisher!

You should write your own plugin, PlotJuggler does not provide any useful publisher out of the box.

If you need commercial support to do that, let me know.
